### ISSUE TITLE
Deduplicate completion entries for aliased providers

### ISSUE THREAD

**Issue by radeksimko** [hashicorp/terraform-ls#439]
### Server Version

```
v0.14.0
```

### Terraform Configuration Files

```hcl
terraform {
  required_providers {
    aws = {
      source  = "hashicorp/aws"
      version = "~> 3.29"
    }
  }
}

provider "aws" {
  alias = "blah"
}

resource "" "name" {
  
}
```

### Expected Behavior

See each `aws` resource type exactly once.

### Actual Behavior

![Screenshot 2021-03-12 at 08 58 25](https://user-images.githubusercontent.com/287584/110916990-4a599680-8311-11eb-8bdf-a58258fab54a.png)

### Steps to Reproduce

1. Request completion within the first label of `resource`

**Comment by radeksimko** [comment]
This is being addressed as part of https://github.com/hashicorp/hcl-lang/pull/32

**Comment by radeksimko** [comment]
Addressed via #454 which brought the updated `hcl-lang` with the linked patch.
